docs/bib/use_citations.bib

```diff
@@ -9,6 +9,34 @@ @article{Alguero:2020grj
     journal = ""
 }
 
+% 2020-06-20
+@article{Krupa:2020bwg,
+    author = "Krupa, Jeffrey and others",
+    title = "{GPU coprocessors as a service for deep learning inference in high energy physics}",
+    eprint = "2007.10359",
+    archivePrefix = "arXiv",
+    primaryClass = "physics.comp-ph",
+    reportNumber = "FERMILAB-PUB-20-338-E-SCD",
+    month = "7",
+    year = "2020",
+    journal = ""
+}
```

Comment:
It is worth noting that this paper cites `pyhf` wrong. They clearly are citing `pyhf` but then give the HistFactory paper citation instead. I'd still put this up for now even though we should email them to ask for a correction.